### PR TITLE
test: should not send email for non-existing email

### PR DESCRIPTION
**Pull request by danielgent**
Sadly these tests pass :see_no_evil:

### CODE REVIEW

File: test/integration/login/forgot-password.spec.ts [hunk 1]
```diff
@@ -73,7 +76,98 @@ describe("Forgot password", () => {
...
+    await snapshotResponse(forgotPasswordEmailResponse);
+
+    // ---------------------
+    // no email has been sent
+    // ---------------------
+
+    expect(env.data.emails).toHaveLength(0);
+  });
+
+  it("should not send a forgot password email for a non-password user with that email address", async () => {
```
Comment (danielgent) [inline test/integration/login/forgot-password.spec.ts:119]: I'm just testing what is currently happening. No opinion given :smile:

File: test/integration/login/forgot-password.spec.ts [hunk 2]
```diff
@@ -73,7 +76,98 @@ describe("Forgot password", () => {
...
+      query: {
+        state: query.state,
+      },
+      form: {
+        username: "existing-email-user@example.com",
+      },
+    });
+
+    expect(forgotPasswordEmailResponse.status).toBe(200);
+    await snapshotResponse(forgotPasswordEmailResponse);
```
Comment (danielgent) [inline test/integration/login/forgot-password.spec.ts:165]: Do we want a success message here? Do we want to create a new username-password account where the email is already verified? :smile:  :see_no_evil: 

Do I not think about this until we've released the universal auth code flow everywhere? :partying_face:

File: test/integration/login/forgot-password.spec.ts [hunk 3]
```diff
@@ -73,7 +76,98 @@ describe("Forgot password", () => {
...
+
+    // ---------------------
+    // no email has been sent
+    // ---------------------
+
+    expect(env.data.emails).toHaveLength(0);
```
Comment (danielgent) [inline test/integration/login/forgot-password.spec.ts:116]: :see_no_evil: This isn't the answer that you want Markus :see_no_evil:

File: test/integration/login/forgot-password.spec.ts [hunk 4]
```diff
@@ -8,20 +8,22 @@ import {
 } from "../helpers/playwrightSnapshots";
 import { AuthorizationResponseType } from "../../../src/types";
 
+const DEFAULT_AUTHORIZE_PARAMS = {
+  client_id: "clientId",
+  response_type: AuthorizationResponseType.TOKEN_ID_TOKEN,
+  scope: "openid",
+  redirect_uri: "http://localhost:3000/callback",
+  state: "state",
+};
```
Comment (danielgent) [inline test/integration/login/forgot-password.spec.ts:17]: maybe we want a helper that does this first `/authorize` step. Maybe we could directly create a `universal_login_session` (I've done that on some tests)

**Review by markusahlstrand — APPROVED** [review]
Looks good!

**Pull request merged**